flashinfer_bench/apply/runtime.py

```
@@ -210,9 +222,42 @@ def dispatch(
+        runtime (if any) is restored.
+
+        Raises
+        ------
+        RuntimeError
+            If this runtime is not the currently active instance.
+        """
+        if not ApplyRuntime._stack or ApplyRuntime._stack[-1] is not self:
+            raise RuntimeError(
+                "Cannot stop an ApplyRuntime that is not the current active instance. "
+                "Runtimes must be stopped in LIFO order."
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)